index.html

```diff
@@ -31,16 +34,192 @@ <h1>Melanie Cascante</h1>
     </section>
 
     <div class="box">
-        <a href="#">
-            <i class="Start_collab"></i>
+        <a href="#" class="button1">
+            <i class="Start_collab"></i>Start Collaboration</a>
     </div>
 
-    <div class="start_box">Start collaboration</div>
-
-
     <section class="geometrybottom">
         <img src="images/image_geometry_2.png">
     </section>
+
+    <section class="projects">
+        <h2>Projects</h2>  
+    </section>
+
+</section>
+<div class="rectangle21v">
+    <img src="images/Rectangle 21.png">
+</div>
+
+<section class="works">
+    <header>
+        <h3>Hello Microverse!</h3>
+     </header>   
+      <ul>
+        <li><div class="item1">Html</div></li>
+        <li><div class="item2">Css</div></li> 
+        <li><div class="item3">JavaScript</div></li>
```

Comment:
As the previous reviewer highlighted, try and make your code match the figma template. If you look at this list items, they are centered on the template, but in your case they look uneven.
This is the figma design:
<img width="321" alt="Screenshot 2023-01-20 at 09 42 03" src="https://user-images.githubusercontent.com/48546992/213636586-19793daa-17fc-4bd1-9ede-ab02ce4258c0.png">
This is your design:
<img width="378" alt="Screenshot 2023-01-20 at 10 07 29" src="https://user-images.githubusercontent.com/48546992/213637657-7b77b826-818d-410e-9051-2b4faefbfcd1.png">
You can get rid of the margins you have on this items, the `margin right: 15px ` and instead arrange the items using `display :flex` and use then `gap` property to give space between items. The margins are causing a bug in your code.

Below i have changed `class="item1"` html item just as a demo of what you can do to get consistency on the items:
<img width="1608" alt="Screenshot 2023-01-20 at 10 05 05" src="https://user-images.githubusercontent.com/48546992/213638127-8d539651-3c27-4af7-9da3-b9de73c637e5.png">